manifests/cf-manifest/operations.d/720-rds-broker-postgres12-plans.yml

```diff
@@ -91,9 +91,12 @@
     medium_plan_rds_properties: &medium_plan_rds_properties
       db_instance_class: "db.m5.large"
       allocated_storage: 100
-    large_plan_rds_properties: &large_plan_rds_properties
+    old_large_plan_rds_properties: &old_large_plan_rds_properties
```

Comment:
Should we change this to be `deprecated_large_plan_rds_properties` to make it linguistically consistent

Reply:
Going to keep this as old to be consistent with the old_small stuff, and change `-deprecated` to `-old` in the names